features/gba-dev/verblijfstitel.feature

```diff
@@ -0,0 +1,91 @@
+
+@post-assert
+Functionaliteit: Verblijfstitel
+
+Achtergrond:
+  Gegeven landelijke tabel "Verblijfstitel" heeft de volgende waarden
+  | code | omschrijving                                                                   |
+  |   00 | Onbekend                                                                       |
+  |   09 | Art. 9 van de Vreemdelingenwet                                                 |
+  |   37 | Vw 2000 art. 8, onder e, gemeenschapsonderdaan econ. niet-actief, arbeid spec. |
+
+  Scenario: verblijfstitel heeft geen verblijfstitel
+    Gegeven de persoon met burgerservicenummer '000000103' heeft de volgende gegevens
+    | naam                  | waarde |
+    | geslachtsnaam (02.40) | Jansen |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000103                       |
+    | fields              | verblijfstitel.aanduiding       |
+    Dan heeft de response een persoon zonder gegevens
+
+  Scenario: verblijfstitel heeft geen datum einde
+    Gegeven de persoon met burgerservicenummer '000000103' heeft de volgende 'verblijfstitel' gegevens
+    | aanduiding verblijfstitel (39.10) | datum ingang verblijfstitel (39.30) |
+    | 37                                | 20210315                            |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000103                       |
+    | fields              | verblijfstitel.aanduiding       |
+    Dan heeft de response een persoon met alleen de volgende 'verblijfstitel' gegevens
+    | naam                    | waarde                                                                              |
+    | aanduiding.code         | 37                                                                                  |
+    | aanduiding.omschrijving | Vw 2000 art. 8, onder e, gemeenschapsonderdaan econ. niet-actief, arbeid spec. |
+
+  Scenario: verblijfstitel heeft een datum einde
+    Gegeven de persoon met burgerservicenummer '000000140' heeft de volgende 'verblijfstitel' gegevens
+    | aanduiding verblijfstitel (39.10) | datum ingang verblijfstitel (39.30) | datum einde verblijfstitel (39.20) |
+    | 37                                | 19980201                            | 20020701                            |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000140                       |
+    | fields              | verblijfstitel.aanduiding       |
+    Dan heeft de response een persoon zonder gegevens
+
+  Scenario: persoon's verblijfstitel velden is in onderzoek
```

Comment:
In dit geval is de verblijfstitel toch alleen in onderzoek geweest, en niet in onderzoek?
Of: onderzoek naar persoon's verblijfstitelgegevens is afgerond?

Reply:
titel snap ik niet en lijkt niet overeen te komen met de Gegeven stappen.

Gegeven zegt dat de persoon geen verblijfstitel heeft, en dat het onderzoek is beëindigd. Lijkt de combinatie van twee dingen te testen:
1. niks leveren wanneer gevraagd gegeven geen waarde heeft
2. niet leveren beëindigd onderzoek (en dan ook geen leeg object verblijfstitel)

Reply:
Zodanig aangepast dat alleen wordt getest dat een beëindigd onderzoek niet wordt geleverd. (en de titel daar ook op aangepast)